linux_os/guide/auditing/auditd_configure_rules/directory_access_var_log_audit/kubernetes/shared.yml

```diff
@@ -6,6 +6,6 @@
 # disruption = medium
 #
 {{% macro rhcos_access_var_log_audit_rules() -%}}
--a always,exit -F dir=/var/log/audit/ -F perm=r -F auid>={{{ auid }}} -F auid!=unset -F key=access-audit-trail
+-a always,exit -F arch=b64 -F dir=/var/log/audit/ -F perm=r -F auid>={{{ auid }}} -F auid!=unset -F key=access-audit-trail
```

Comment:
Need to investigate if there is a way to set this to 32 or 64 at run time, instead of hardcoding it.

Reply:
Actually - we need both of these so that we audit both 64 and 32 bit applications running on top of the nodes.